packages/project-management-automation/lib/tasks/first-time-contributor-account-link/test/index.js

```diff
@@ -32,6 +32,36 @@ describe( 'firstTimeContributorAccountLink', () => {
 		},
 	};
 
+	it( 'does nothing for commits by bots', async () => {
+		const payloadForBot = {
+			...payload,
+			commits: [
+				{
+					id: '4c535288a6a2b75ff23ee96c75f7d9877e919241',
+					message: 'Add a feature from pull request (#123)',
+					author: {
+						name: 'Ghost',
+						email: 'ghost@example.invalid',
+						username: 'ghost',
+						type: 'Bot',
```

Comment:
Is there a way of manually testing this? I dug around a bit and I think the webhook push event only includes email, name and email address for the author object: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push

If it happens to not be included, I don't think it'd be too bad making a secondary rest request. Code makes sense otherwise 👍 

<img width="593" alt="CleanShot 2022-02-01 at 12 34 07@2x" src="https://user-images.githubusercontent.com/1270189/152047283-8b2955c8-a2fc-408e-87f8-4f900dbe5371.png">

Reply:
Thanks for finding that. I guess this wouldn't have worked for that particular job, and there's a strong chance I would have been scratching my head for a while trying to find out why!

I suppose I can test this out on a fork to double check the payload—it doesn't have to be a bot after all, as long as I can determine whether `type` is there.

Reply:
Yep, commit author definitely only has three fields. I've updated the code now, so this should be fixed.